content/en/docs/ibm/using-icr.md

```diff
@@ -28,20 +26,20 @@ kubectl get secret all-icr-io -n default -o yaml \
 | kubectl -n anonymous create -f -
 ```
 
-Once this secret is ready in your Kubeflow profile, A data scientist can use this secret to pull container images from ICR.
+Once this secret is ready in your Kubeflow profile, a data scientist can use this secret to pull container images from ICR.
```

Comment:
To avoid repetition ("this... this..."):

>... a data scientist can use the secret

or better - replace with "it":

>... a data scientist can use it ....